### ISSUE TITLE
Replace imp package in runtests.py

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#174]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/174) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

We get the following Deprecation when running tests.

```
runtests.py:1: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
```

Make this go away. Also, figure out how to _not_ search through the `.git` directories for tests.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

You can `import importlib` \(instead of imp\), and try this patch:

```
- modules.append(imp.load_module(module_name, *imp.find_module(module_name)))
+ modules.append(importlib.import_module(module_name))
```

But as far as I can tell, it breaks the build on Python 2.7 and slows down tests on Python 3.7. I think I’ll put this issue on hold.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Can't figure out a solution that works well on Python 2 and 3. @{557058:9524f6a0-ef9f-4466-b913-d04e51ff9093} suggests a simpler loop over files, which is fine. But this isn't worth any more time than I already put into it.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)

**Comment by joefowler** [comment]
This was fixed by the time of [version 0.7.3](https://github.com/usnistgov/mass/releases/tag/versions%2F0.7.3) in March 2020.
